src/Control/Middleware/HTTPCacheControlMiddleware.php

```diff
@@ -238,7 +238,7 @@ public function setVary($vary)
     /**
      * Combine vary strings/arrays into a single array, or normalise a single vary
      *
-     * @param string|array[] $varies Each vary as a separate arg
+     * @param string|array ... $varies Each vary as a separate arg
```

Comment:
Do you happen to have a reference for this change? As far as I know PHPDoc (and PSR-5/19) don't have a standard yet for variadic arguments